src/itn_uptime_analyzer/uptime.go

```diff
@@ -82,56 +81,65 @@ func (identity Identity) GetUptime(config AppConfig, sheet *sheets.Service, ctx
 
 					if submissionDataToday.GraphqlControlPort != 0 {
 						if (identity.publicKey == submissionDataToday.Submitter.String()) && (identity.publicIp == submissionDataToday.RemoteAddr) && (*identity.graphQLPort == strconv.Itoa(submissionDataToday.GraphqlControlPort)) {
-							if lastSubmissionTimeString != "" {
-								lastSubmissionTime, err = time.Parse(time.RFC3339, lastSubmissionTimeString)
-								if err != nil {
-									log.Fatalf("Error parsing time: %v\n", err)
-								}
-							}
 
 							currentSubmissionTime, err := time.Parse(time.RFC3339, submissionDataToday.CreatedAt)
 							if err != nil {
 								log.Fatalf("Error parsing time: %v\n", err)
 							}
 
-							if (lastSubmissionTimeString != "") && (currentSubmissionTime.After(lastSubmissionTime.Add(time.Duration(syncPeriod-5) * time.Minute))) && (currentSubmissionTime.Before(lastSubmissionTime.Add(time.Duration(syncPeriod+5) * time.Minute))) {
-								uptimeYesterday = append(uptimeYesterday, true)
+							if lastSubmissionTimeString != "" {
+								lastSubmissionTime, err = time.Parse(time.RFC3339, lastSubmissionTimeString)
+								if err != nil {
+									log.Fatalf("Error parsing time: %v\n", err)
```

Comment:
When calling `log.Fatal` or `log.Fatalf` you don't need to enclose what comes next in `else`, since they call `os.Exit` and execution halts there.